### ISSUE TITLE
OValidationException when moving the DB from 1.7.x to 2.0.1

### ISSUE THREAD

**Issue by nooruls143** [orientechnologies/orientdb#3547]
Hi,
I exported my Db from version 1.7.x successfully using the guide http://www.orientechnologies.com/docs/last/orientdb.wiki/Migration-from-1.7.x-to-2.0.x.html and when I am trying to import it in the version 2.0.1, I get the following error:

```
Creating database [remote:localhost/pricewiz] using the storage type [remote]...
Connecting to database [remote:localhost/pricewiz] with user 'admin'...OK
Database created successfully.

Current database is: remote:localhost/pricewiz
orientdb {db=pricewiz}> import database /tmp/pricewiz.json.gz

Importing database database /tmp/pricewiz.json.gz...
Started import of database 'remote:localhost/pricewiz' from /tmp/pricewiz.json.gz...
Non merge mode (-merge=false): removing all default non security classes
2015-02-07 09:55:21:120 SEVERE Removing disconnected network channel '127.0.0.1:2424/pricewiz'... [OStorageRemote]
2015-02-07 09:55:21:275 WARNING Caught I/O errors from Not connected (local socket=?), trying to reconnect (error: java.io.IOException: Channel is closed) [OStorageRemote]
2015-02-07 09:55:23:407 WARNING Connection re-acquired transparently after 2120ms and 1 retries to server '127.0.0.1:2424/pricewiz': no errors will be thrown at application level [OStorageRemote]
- Class E was removed.
- Class V was removed.
- Class ORestricted was removed.
- Class OTriggered was removed.
- Class OSchedule was removed.
- Class ORIDs was removed.
- Class OFunction was removed.
Removed 7 classes.
Importing database info...OK
Importing clusters...
- Creating cluster 'internal'...OK, assigned id=0
- Creating cluster 'default'...OK, assigned id=3
- Creating cluster 'orole'...OK, assigned id=4
- Creating cluster 'ouser'...OK, assigned id=5
- Creating cluster 'orids'...OK, assigned id=6
- Creating cluster 'ofunction'...OK, assigned id=7
- Creating cluster 'oschedule'...OK, assigned id=8
- Creating cluster 'v'...OK, assigned id=9
- Creating cluster 'e'...OK, assigned id=10
- Creating cluster 'category'...OK, assigned id=11
- Creating cluster '_studio_bookmark'...OK, assigned id=12
- Creating cluster 'product'...OK, assigned id=13
- Creating cluster 'price'...OK, assigned id=14
- Creating cluster 'offer'...OK, assigned id=15
- Creating cluster 'store'...OK, assigned id=16
- Creating cluster 'inf_user'...OK, assigned id=17
- Creating cluster 'recentproducts'...OK, assigned id=18
- Creating cluster 'inf_recentproducts'...OK, assigned id=19
Rebuilding indexes of truncated clusters ...
Done 2 indexes were rebuilt.
Done. Imported 18 clusters
Importing database schema...OK (18 classes)

Importing records...
2015-02-07 10:01:04:730 SEVERE Error importing record #-1:-1. Source line 0, column 62930 [ODatabaseImport]Error on database import happened just before line 0, column 62931
com.orientechnologies.orient.enterprise.channel.binary.OResponseProcessingException: Exception during response processing.
    at com.orientechnologies.orient.enterprise.channel.binary.OChannelBinaryAsynchClient.throwSerializedException(OChannelBinaryAsynchClient.java:441)
    at com.orientechnologies.orient.enterprise.channel.binary.OChannelBinaryAsynchClient.handleStatus(OChannelBinaryAsynchClient.java:396)
    at com.orientechnologies.orient.enterprise.channel.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:282)
    at com.orientechnologies.orient.enterprise.channel.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:171)
    at com.orientechnologies.orient.client.remote.OStorageRemote.beginResponse(OStorageRemote.java:1865)
    at com.orientechnologies.orient.client.remote.OStorageRemote.createRecord(OStorageRemote.java:357)
    at com.orientechnologies.orient.client.remote.OStorageRemoteThread.createRecord(OStorageRemoteThread.java:243)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.executeSaveRecord(ODatabaseDocumentTx.java:1713)
    at com.orientechnologies.orient.core.tx.OTransactionNoTx.saveRecord(OTransactionNoTx.java:94)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.save(ODatabaseDocumentTx.java:2271)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.save(ODatabaseDocumentTx.java:117)
    at com.orientechnologies.orient.core.record.impl.ODocument.save(ODocument.java:1747)
    at com.orientechnologies.orient.core.record.impl.ODocument.save(ODocument.java:1738)
    at com.orientechnologies.orient.core.record.impl.ODocument.save(ODocument.java:75)
    at com.orientechnologies.orient.core.db.tool.ODatabaseImport.importRecord(ODatabaseImport.java:1291)
    at com.orientechnologies.orient.core.db.tool.ODatabaseImport.importRecords(ODatabaseImport.java:1187)
    at com.orientechnologies.orient.core.db.tool.ODatabaseImport.importDatabase(ODatabaseImport.java:425)
    at com.orientechnologies.orient.console.OConsoleDatabaseApp.importDatabase(OConsoleDatabaseApp.java:1538)
    at com.orientechnologies.orient.graph.console.OGremlinConsole.importDatabase(OGremlinConsole.java:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.orientechnologies.common.console.OConsoleApplication.execute(OConsoleApplication.java:381)
    at com.orientechnologies.common.console.OConsoleApplication.executeCommands(OConsoleApplication.java:246)
    at com.orientechnologies.common.console.OConsoleApplication.run(OConsoleApplication.java:135)
    at com.orientechnologies.orient.graph.console.OGremlinConsole.main(OGremlinConsole.java:62)
Caused by: com.orientechnologies.orient.core.exception.OValidationException: The field 'Dic_Category.category_id' has been declared as LINKSET but contains a null record (probably a deleted record?)
    at com.orientechnologies.orient.core.record.impl.ODocument.validateLink(ODocument.java:451)
    at com.orientechnologies.orient.core.record.impl.ODocument.validateLinkCollection(ODocument.java:435)
    at com.orientechnologies.orient.core.record.impl.ODocument.validateField(ODocument.java:286)
    at com.orientechnologies.orient.core.record.impl.ODocument.validate(ODocument.java:1985)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.save(ODatabaseDocumentTx.java:2215)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.save(ODatabaseDocumentTx.java:2108)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.save(ODatabaseDocumentTx.java:117)
    at com.orientechnologies.orient.server.network.protocol.binary.OBinaryNetworkProtocolAbstract.createRecord(OBinaryNetworkProtocolAbstract.java:349)
    at com.orientechnologies.orient.server.network.protocol.binary.ONetworkProtocolBinary.createRecord(ONetworkProtocolBinary.java:1375)
    at com.orientechnologies.orient.server.network.protocol.binary.ONetworkProtocolBinary.executeRequest(ONetworkProtocolBinary.java:350)
    at com.orientechnologies.orient.server.network.protocol.binary.OBinaryNetworkProtocolAbstract.execute(OBinaryNetworkProtocolAbstract.java:216)
    at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:65)

Error: com.orientechnologies.orient.core.db.tool.ODatabaseExportException: Error on importing database 'pricewiz' from file: /tmp/pricewiz.json.gz

Error: com.orientechnologies.orient.enterprise.channel.binary.OResponseProcessingException: Exception during response processing.

Error: com.orientechnologies.orient.core.exception.OValidationException: The field 'Dic_Category.category_id' has been declared as LINKSET but contains a null record (probably a deleted record?)
```

Please could you check this issue.
Regards,
Noor

**Comment by lvca** [comment]
It's a known bug if you import to a server. Please stop the server and use 'plocal' connection from console.

**Comment by santo-it** [comment]
Hi,

Thanks again for opening this issue. Fixed in 2.2.  Closing
